### PR TITLE
Admins for locations && cohorts

### PR DESCRIPTION
**Pull request by nolds9**

### CODE REVIEW

**Comment by nolds9** [comment]
Note: no UI to create associations for Admins, potentially move to seed data

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
 
+  # TODO: Refactor authorization to use before_filters || helpers
+
   def index
     @courses = Course.all
   end
 
   def show
+    authorize! :manage, @course
   end
```
Comment (amaseda) [inline app/controllers/courses_controller.rb]: replace :manage with show

File: app/models/course.rb [hunk 2]
```diff
@@ -1,3 +1,8 @@
 class Course < ActiveRecord::Base
   FORMATS = ["Immersive", "Course"]
+  has_and_belongs_to_many :users
+
```
Comment (amaseda) [inline app/models/course.rb]: change to has admins as users

File: db/migrate/20160127194910_create_courses_users.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class CreateCoursesUsers < ActiveRecord::Migration
+  def change
+    create_table :courses_users do |t|
+      t.references :course
```
Comment (amaseda) [inline db/migrate/20160127194910_create_courses_users.rb]: create_join_table

File: app/controllers/courses_controller.rb (hunk 1 above)
Comment (amaseda) [inline app/controllers/courses_controller.rb]: :+1:

**Comment by amaseda** [comment]
addressed by https://github.com/ga-dc/garnet/pull/359 with passing build

**Pull request closed without merging**